### DISCUSSION TITLE
Version per component

### DISCUSSION THREAD

**Discussion by LucasCaixeta** [Gympass/yoga#467] [General]
Do you have version per component? or a version for the entire project?

I mean, if the button is upgraded, will we have a new version for the button or a new version for the entire project?

**Comment by mmartinsoliv** [comment]
Hi @LucasCaixeta , sorry for the delay. any kind of changes made to a package, the package will be updated.

For example, the button is part of the `@gympass/yoga` package, if a change is made to this button, the `@gympass/yoga` package will generate a new version
